backend/scripts/seed_data.py

```diff
@@ -165,51 +169,91 @@
-                status=C3TestResultStatus.FAIL,
-                category="",
-                comment="",
-                io_log="",
-            ),
-            C3TestResult(
-                id=2,
-                name="test2",
+                name="docker/compose-and-basic_armhf",
                 status=C3TestResultStatus.PASS,
-                category="",
+                category="Docker containers",
                 comment="",
-                io_log="",
+                io_log=dedent(
+                    """
+                    test Pulling 
+                    test Pulled 
+                    Network 4506_default  Creating
+                    Network 4506_default  Created
+                    Container 4506-test-1  Creating
+                    Container 4506-test-1  Created
+                    Container 4506-test-1  Starting
+                    Container 4506-test-1  Started
+                    Container 4506-test-1  Stopping
+                    Container 4506-test-1  Stopped
+                    Container 4506-test-3  Creating
+                    Container 4506-test-2  Creating
+                    Container 4506-test-1  Recreate
+                    Container 4506-test-3  Created
+                    Container 4506-test-2  Created
+                    Container 4506-test-1  Recreated
+                    Container 4506-test-1  Starting
+                    Container 4506-test-1  Started
+                    Container 4506-test-2  Starting
+                    Container 4506-test-2  Started
+                    Container 4506-test-3  Starting
+                    Container 4506-test-3  Started
+                    3
+                    Container 4506-test-1  Running
+                    Container 4506-test-1  Stopping
+                    Container 4506-test-1  Stopping
+                    Container 4506-test-1  Stopped
+                    Container 4506-test-1  Removing
+                    Container 4506-test-1  Removed
+                    Network 4506_default  Removing
+                    Network 4506_default  Removed
+                    """
+                ),
             ),
             C3TestResult(
-                id=3,
-                name="test3",
+                id=2,
+                name="after-suspend-audio/alsa-loopback-automated",
                 status=C3TestResultStatus.SKIP,
-                category="",
-                comment="",
+                category="Audio tests",
+                comment="""job cannot be started: resource expression "manifest.has_audio_loopback_connector == 'True'" evaluates to false, required dependency 'com.canonical.certification::audio/alsa-loopback-automated' has failed, required dependency 'com.canonical.certification::audio/detect-capture-devices' has failed, required dependency 'com.canonical.certification::audio/detect-playback-devices' has failed, required dependency 'com.canonical.certification::suspend/suspend_advanced_auto' has failed""",
```

Comment:
are you sure you don't want to split this string?

Reply:
Tried to, including the other long lines, but it got uglier and more complicated :/